docs/reference/networking.md

```diff
@@ -0,0 +1,16 @@
+# 5G Network Architecture
+
+5G was built with network functions divided by services which is known as the 5G core Service-Based Architecture (SBA). The 5G core is a network of interconnected services, as illustrated in the figure below.
+
+```{image} ../images/5g_network_architecture.png
+:alt: 5G Network Architecture
+:height: 600
+:align: center
+```
+
```

Comment:
We wanted to include a table of ports too, I think we're missing this. @ghislainbourgeois do you know what those ports/directions are?

Reply:
Added ports belongs to AMF and UPF as table.